automation/taskcluster/helper/get-secret.py

```diff
@@ -21,7 +22,12 @@ def write_secret_to_file(path, data, key, base64decode=False, append=False, pref
         value = data['secret'][key]
         if base64decode:
             value = base64.b64decode(value)
-        f.write(prefix + value)
+
+        try:
+            json_string = json.dumps(value)
+            f.write(prefix + json_string)
+        except ValueError as e:
+            f.write(prefix + value)
```

Comment:
Stop! This breaks secrets that are not JSON and we end up JSON encoding it. We investigated why RB nightlies stopped building and that's because some token was written as JSON now and instead of `token` the file contained `"token"`. Please port this patch from RB over:
https://github.com/mozilla-mobile/reference-browser/commit/4b2c3b4dcfdc885e400e838d74e8ea7740ab194b

Reply:
yikes, I thought I had implemented it in such a way that it would handle both, but apparently not.  Apologies for the inconvenience caused.  Anyway, patch applied